src/components/About/About.tsx

```diff
@@ -1,159 +1,153 @@
+            </Box>
+            <Box sx={paragraphStyle}>
+                <Typography>
                     <Trans
-                        i18nKey={"about:firstParagraph"}
+                        i18nKey="about:firstParagraph"
                         components={[
                             <a
                                 style={{ whiteSpace: "pre-wrap" }}
                                 href="https://github.com/AletheiaFact/aletheia"
                                 target="_blank"
                                 rel="noreferrer"
-                            ></a>,
+                            />,
                             <a
                                 style={{ whiteSpace: "pre-wrap" }}
                                 href="https://builders.mozilla.community/programs.html#open-lab"
                                 target="_blank"
                                 rel="noreferrer"
-                            ></a>,
+                            />,
                         ]}
                     />
-                </p>
-            </Row>
-            <Row style={paragraphStyle}>
-                <p>
+                </Typography>
+            </Box>
+            <Box sx={paragraphStyle}>
+                <Typography>
                     <Trans
-                        i18nKey={"about:secondParagraph"}
+                        i18nKey="about:secondParagraph"
                         components={[
                             <a
                                 style={{ whiteSpace: "pre-wrap" }}
                                 href="https://wikipedia.org"
                                 target="_blank"
                                 rel="noreferrer"
-                            ></a>,
+                            />,
                             <a
                                 style={{ whiteSpace: "pre-wrap" }}
                                 href="https://demagog.cz/"
                                 target="_blank"
                                 rel="noreferrer"
-                            ></a>,
+                            />,
                         ]}
                     />
-                </p>
-            </Row>
-            <Row style={paragraphStyle}>
-                <p>
-                    <Trans i18nKey={"about:thirdParagraph"} />
-                </p>
-            </Row>
-            <Row style={paragraphStyle}>
-                <p>
-                    <Trans i18nKey={"about:forthParagraph"} />
-                </p>
-            </Row>
-        </Row>
+                </Typography>
+            </Box>
+            <Box sx={paragraphStyle}>
+                <Typography>
+                    <Trans i18nKey="about:thirdParagraph" />
+                </Typography>
+            </Box>
+            <Box sx={paragraphStyle}>
+                <Typography>
+                    <Trans i18nKey="about:forthParagraph" />
+                </Typography>
```

Comment:
question: why changing the p paragraph to Typography?

Reply:
The change from p to Typography was made to ensure a standardization of MUI theme styling.